src/Security/Security/help/Get-AzSecurityThreatProtection.md

```diff
@@ -0,0 +1,79 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Security.dll-Help.xml
+Module Name: Az.Security
+online version: https://docs.microsoft.com/en-us/powershell/module/az.security/get-azsecuritythreatprotection
+schema: 2.0.0
+---
+
+# Get-AzSecurityThreatProtection
+
+## SYNOPSIS
+Gets the threat protection policy for a storage account.
+
+## SYNTAX
+
+```
+Get-AzSecurityThreatProtection -ResourceId <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The "Get-AzSecurityThreatProtection" cmdlet gets the threat protetion policy of a storage account.
+To use this cmdlet, specify the *ResourceId* parameter.
+
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzSecurityThreatProtection -ResourceId "/subscriptions/cma24pc8-89b5-4aa7-9ff6-486e886c304a/resourceGroups/myResourceGroup/providers/Microsoft.Storage/storageAccounts/myStorageAccount/"
```

Comment:
Please add example output for all examples

Reply:
Please also remove the subscription and replace with xxxxxxx-xxxx-xxxx-xxxxxxxxxxxxx